packages/fluentui/accessibility/src/behaviors/Pill/pillOptionBehavior.ts

```diff
@@ -0,0 +1,28 @@
+import { Accessibility, AriaRole } from '../../types';
+import { keyboardKey } from '../../keyboard-key';
+import { IS_FOCUSABLE_ATTRIBUTE } from '../../attributes';
+
+export const pillOptionBehavior: Accessibility<PillBehaviorProps> = p => ({
+  attributes: {
+    root: {
+      role: 'option',
+      [IS_FOCUSABLE_ATTRIBUTE]: true,
+    },
+  },
+  keyActions: {
+    root: {
+      performDismiss: {
+        keyCombinations: [
+          { keyCode: keyboardKey.Escape },
```

Comment:
Esc is not used for removing, was this specified somewhere? If yes we will need to change the spec.

Reply:
My bad, removed